skillserver/skillserver.go

```diff
@@ -199,16 +206,10 @@ func verifyJSON(w http.ResponseWriter, r *http.Request, next http.HandlerFunc) {
 
 // Run all mandatory Amazon security checks on the request.
 func validateRequest(w http.ResponseWriter, r *http.Request, next http.HandlerFunc) {
-	// Check for debug bypass flag
-	devFlag := r.URL.Query().Get("_dev")
```

Comment:
Why are we removing this check? How do we know no one is using it?

Maybe we could move this into `IsValidAlexaRequest`?

Reply:
I get the desire but what if we have users who are actively using this flag? We should preserve it for backwards compatibility.

Reply:
Ok, so i will put a or condition then.
Thanks for the review